data/campaigns/Dead_Water/scenarios/01_Invasion.cfg

```diff
@@ -347,12 +347,12 @@
 
         [message]
             speaker=Gwabbo
-            message= _ "Kai, please lead us against them yourself, it will hearten the folk. And — if you will permit — you reach the age now where a kai must learn battle skills."
+            message= _ "Kai, please lead us against them yourself, it will hearten the folk. And — if you will permit — you have now reached the age where a kai must learn battle skills."
         [/message]
 
         [message]
             speaker=Cylanna
-            message= _ "Gwabbo speaks wisely. You must fight; I will protect you as much as I can...but when war looms, a kai must lead. It is our way."
+            message= _ "Gwabbo speaks wisely. You must fight! I will protect you as much as I can...but when war looms, a kai must lead. It is our way."
```

Comment:
I don't like this change. I think the semicolon instead of an exclamation mark may be intended to make paint her as a calmer sort of person.